src/runtime/math.py

```diff
@@ -296,3 +296,42 @@ def _lfortran_datanh(x: f64) -> f64:
 
 def atanh(x: f64) -> f64:
     return _lfortran_datanh(x)
+
+
+def ceil(x: f64) -> i32:
+    if x > 0 and fmod(x, 1.0) != 0:
+        return int(x)+1
+    return int(x)
+
+
+def floor(x: f64) -> i32:
+    if x < 0 and fmod(x, 1.0) != 0:
+        return int(x)-1
+    return int(x)
```

Comment:
Why is the `fmod` part needed? LFortran has this implementation of `floor`: https://gitlab.com/lfortran/lfortran/-/blob/28fa2448c6a1c950d2eb7ca1ecc8f0c9b84293b1/src/runtime/pure/lfortran_intrinsic_math2.f90#L135

Reply:
In condition ` if x < 0 and fmod(x, 1.0) != 0:`,  for floor calculation if the number, say x is negative and has non zero fractional part it should return int(x)-1. `fmod(x, 1.0)` is required to compute the fractional part.